### DISCUSSION TITLE
Multiple authors in a blog not showing up

### DISCUSSION THREAD

**Discussion by pranay01** [facebook/docusaurus#6200] [Q&A] [answered]
I am on docusaurus beta.14
I want to show multiplw authors for some of my blogs. But when add authors frontmatter - none of the authors show up. Do I need to enable this someplace else also?

Here's the PR I am trying - https://github.com/SigNoz/signoz.io/pull/133

Page where multiple authors should have shown up - https://deploy-preview-133--frosty-austin-928e43.netlify.app/blog/opentelemetry-nestjs/

Any ideas on what am I doing wrong?

**Comment by Josh-Cena** [comment, marked as answer]
Hi, your swizzled blog post item does not respect the `authors` front matter:

https://github.com/SigNoz/signoz.io/blob/dd48855efd7ea9064f35c671626659b355436e87/src/theme/BlogPostItem/index.js#L39

Please reswizzle your components related to blog and incorporate our changes

**Reply by pranay01** [reply]
thanks, reswizzling solved the issue
